app/templates/courses/teacher-courses-view.jade

```diff
@@ -86,11 +93,18 @@ mixin course-info(course)
           span.spr ,
         
     if me.isTeacher() || view.ownedClassrooms.size() || me.isAdmin()
-      p
-        a.guide-btn.btn.btn-primary(href=("/teachers/course-solution/" + course.id + "/python") data-course-id=course.id data-course-name=course.get('name') data-event-action="Classes Guides Guide Python" class=(me.isTeacher() || me.isAdmin() ? '': 'disabled'))
+      if course.get('slug') === 'web-dev-1'
+        a.guide-btn.btn.btn-primary(href=("/teachers/course-solution/" + course.id + "/javascript") data-course-id=course.id data-course-name=course.get('name') data-event-action="Classes Guides Guide JavaScript" class=(me.isTeacher() || me.isAdmin() ? '': 'disabled'))
```

Comment:
It looks like this guide should be titled (in the document) as "HTML" or "HTML/CSS"; it has no JavaScript in it. Not sure where that value is filled in from.